drivers/timer/stm32_lptim_timer.c

```diff
@@ -110,8 +110,6 @@ int z_clock_driver_init(struct device *device)
 #endif /* LL_APB1_GRP1_PERIPH_PWR */
 	/* enable backup domain */
 	LL_PWR_EnableBkUpAccess();
```

Comment:
Is power on the backup domain still required then ?

Reply:
Yes it's need to enable LSE clock (LSEON bit in RCC_BDCR which is protect by DBP bit)